### PR TITLE
Init to Win it!

### PR DESCRIPTION
**Pull request by PoweredByPot**
Complete rewrite of the entire script that only touches the order, formatting and commenting of the existing content.
The only occasions where something was actually modified from previous versions were because of duplication issues or other minor errors.

### CODE REVIEW

**Comment by AlexTriada** [comment]
@PoweredByPot! Hi, I also had to get into this file, it just so happened, let's coordinate the work somehow.

**Comment by AlexTriada** [comment]
@PoweredByPot you don't understand me, i fully change this file, too

File: A3-Antistasi/initVar.sqf [hunk 1]
```diff
@@ -1,257 +1,185 @@
-//Antistasi var settings
-//If some setting can be modified it will be commented with a // after it.
-//Make changes at your own risk!!
-//You do not have enough balls to make any modification and after making a Bug report because something is wrong. You don't wanna be there. Believe me.
-//Not commented lines cannot be changed.
-//Don't touch them.
-
+//Original Author: Barbolani //Edited and updated by the Antstasi Community Development Team
```
Comment (Toakan) [inline A3-Antistasi/initVar.sqf]: Can you newline the second comment please?

File: A3-Antistasi/initVar.sqf [hunk 2]
```diff
@@ -1,257 +1,185 @@
...
+//RHS AFRF Detection
+if isClass (configFile >> "CfgFactionClasses" >> "rhs_faction_vdv") then {hasAFRF = true; hasRHS = true; diag_log format ["%1: [Antistasi] | INFO | initVar | RHS AFRF Detected.",servertime];};
+if isClass (configFile >> "CfgFactionClasses" >> "rhs_faction_usarmy") then {hasUSAF = true; hasRHS = true; diag_log format ["%1: [Antistasi] | INFO | initVar | RHS USAF Detected.",servertime];};
+if (hasAFRF && hasUSAF) then {if isClass (configFile >> "CfgFactionClasses" >> "rhsgref_faction_tla") then {activeGREF = true; diag_log format ["%1: [Antistasi] | INFO | initVar | RHS GREF Detected.",servertime];};};
+//3CB Detection
+if (hasAFRF and hasUSAF and activeGREF) then {if ("UK3CB_BAF_L1A1" in arifles) then {has3CB = true; diag_log format ["%1: [Antistasi] | INFO | initVar | 3CB Detected.",servertime];};};
+//FFAA Detection
+if ("ffaa_armas_hkg36k_normal" in arifles) then {hasFFAA = true; diag_log format ["%1: [Antistasi] | INFO | initVar | FFAA Detected.",servertime];};
 
-diag_log format ["%1: [Antistasi] | INFO | initVar | Patching mod weapon support",servertime];
```
Comment (Toakan) [inline A3-Antistasi/initVar.sqf]: Can you throw this back in?

File: A3-Antistasi/initVar.sqf [hunk 3]
```diff
@@ -584,204 +1030,79 @@ timer setVariable [vehCSATPlaneAA,10,true];
...
-unlockedBackpacks = [];
-*/
-
-unlockedOptics = [];
-unlockedMG = [];
-unlockedGL = [];
-unlockedSN = [];
-unlockedAA = [];
 garageIsUsed = false;
+missionsX = []; publicVariable "missionsX";
```
Comment (Toakan) [inline A3-Antistasi/initVar.sqf]: Newline ^

File: A3-Antistasi/initVar.sqf [hunk 4]
```diff
@@ -359,33 +604,32 @@ if !(_typeX in _checked) then
...
+//   WEAPON ATTACHMENTS LIST     ///
+////////////////////////////////////
+diag_log format ["%1: [Antistasi] | INFO | initVar | Creating Weapon Attachment Lists",servertime];
 {
 {
 _item = _x;
 if !(_item in (opticsAAF + flashLights + pointers)) then
 	{
-	if (((_item call BIS_fnc_itemType) select 1) == "AccessorySights") then
+	if (isCLass(configFile >> "CfgWeapons" >> _item >> "ItemInfo" >> "OpticsModes")) then
```
Comment (Spoffy) [inline A3-Antistasi/initVar.sqf]: Need to check if this actually a better check

File: A3-Antistasi/initVar.sqf [hunk 5]
```diff
@@ -1,258 +1,192 @@
...
+// INITIAL SETTING AND VARIABLES ///
+////////////////////////////////////
+diag_log format ["%1: [Antistasi] | INFO | initVar | Setting Initial Variables",servertime];
+debug = false;													//debug variable, not useful for everything..
+diagOn = false;												//Turn on Diag_log messaging (unused - PBP)
+cleantime = 3600;												//time to delete dead bodies, vehicles etc..
+distanceSPWN = 1000;											//initial spawn distance. Less than 1Km makes parked vehicles spawn in your nose while you approach.
+distanceSPWN1 = 1300;											//
+distanceSPWN2 = 500;											//
+musicON = false;												//Extra BGM
```
Comment (Spoffy) [inline A3-Antistasi/initVar.sqf]: Music permanently disabled now?

Comment (PoweredByPot) [reply A3-Antistasi/initVar.sqf]: not permanently, just not forced on for SP

File: A3-Antistasi/initVar.sqf (hunk 4 above)
Comment (PoweredByPot) [reply A3-Antistasi/initVar.sqf]: it returns the exact same results right now.

**Pull request merged**